ui/DesignSystem/Component/EmptyState.svelte

```diff
@@ -29,12 +30,16 @@
     align-items: center;
     height: calc(100vh - var(--bigheader-height));
     text-align: center;
-    max-width: 20rem;
+    max-width: 30rem;
```

Comment:
This would be better defined using vw for improved responsiveness but won't make it a blocker.